gatsby-node.js

```diff
@@ -82,7 +82,9 @@ async function sourcePrimerReactData({actions, createNodeId, createContentDigest
   // Save the current version of Primer React to the GraphQL store.
   // This will be the latest version at the time the site is built.
   // If a new version is released, we'll need to rebuild the site.
-  const {version} = await fetch('https://unpkg.com/@primer/react/package.json').then(res => res.json())
+  const {version} = await fetch(
+    'https://unpkg.com/@primer/react/package.json', {redirect: 'follow'}
```

Comment:
This is causing the deployment build to fail. Maybe try it without following the redirect? I have a feeling the redirect is causing HTML to be fetched instead of JSON.

This is the error I'm seeing in the [Preview check](https://github.com/primer/design/actions/runs/5584528469/jobs/10206064520?pr=521):
> invalid json response body at https://unpkg.com/@primer/react@35.26.1/package.json reason: Unexpected token < in JSON at position 0

Reply:
Oh this was a fix that I merged from master but wasn't merged yet. This is gone now.